primitives/src/config.rs

```diff
@@ -164,8 +164,6 @@ impl ChainInfo {
 /// Precision can differ for the same token from one [`Chain`] to another.
 #[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq, Hash)]
 pub struct TokenInfo {
-    #[deprecated = "we no longer need the sweeper contract & create2 addresses for deposits"]
-    pub min_token_units_for_deposit: BigNum,
```

Comment:
I've made a mistake here. This is not used only for the Create2 deposits but also for the minimum budget of campaigns.
We should keep this configuration and rename it to `min_campaign_budget`.
It should stay in the Token configuration though.